packages/back-end/src/controllers/metrics.ts

```diff
@@ -253,7 +255,7 @@ export async function getMetric(
     });
   }
 
-  const experiments = await ExperimentModel.find(
+  const experiments = await getExperimentsByQuery(
```

Comment:
I'd like to avoid having Mongo queries outside of the Model files.  Here, for example, it's getting the most recent experiments that are using a specific metric and it's only returning summary fields instead of the full interface.  We could create a dedicated function in the Model file for this:

```ts
// Type with only the subset of fields being selected
export type MetricExperimentSummary = Pick<ExperimentInterface, "id", "name", "status", "phases", "results", "analysis">;
export function getRecentExperimentsUsingMetric(metric: MetricInterface): MetricExperimentSummary[] {
  ...
}
```